Source/WebInspectorUI/UserInterface/Views/DetailsSection.css

```diff
@@ -165,39 +165,39 @@ body[dir=rtl] .details-section > .header::before {
 }
 
 .details-section > .content {
-    display: table;
     width: 100%;
-    border-spacing: 0;
-    border-collapse: collapse;
 }
 
 .details-section.collapsed > .content {
     display: none;
 }
 
 .details-section > .content > .group {
-    display: table-row-group;
+    display: grid;
+    grid-template-columns: [label] minmax(85px, max-content) [value] 1fr;
     border-bottom: 1px solid var(--border-color);
 }
 
-.details-section > .content > .group:has(.row.font-variation) {
-    display: unset; /* Unsets `display: table-row-group` because it affects sizing of child elements that are grid containers. */
+.details-section > .content > .group > * {
+    grid-column: 1 / -1;
```

Comment:
This ensures all immediate children span the full width of the grid. 
These are always containers, like `.row`, `.row.simple` in the Fonts details sidebar, or `.event-listener-section` in the Node details sidebar. The containers themselves handle their internal layout: `.row.simple` is now a subgrid.